__tests__/app.test.js

```diff
@@ -148,3 +148,66 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  it("should return the correct value types", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.comments.length > 0).toBe(true);
```

Comment:
Since we know how many comments we expect this article to have, I would check for a specific amount, not just any number above 0.